### PR TITLE
Add active stream with funds to claim

### LINKED ISSUE #3845: Expenditures: As a user I can see a active stream with funds to claim
[Provided by Adrian via Contractors project management platform]

## Description
User has to be able to see a active stream with funds to claim. "Paid to date" and "Available to claim" may vary depends on how many currencies are used in funding source.

## Acceptance Criteria

- an option to see an "Active" state
- an option to claim funds by clicking "Claim funds" button
- an option to see "Paid to date" funds and currencies
- an option to see "Available to claim" funds and currencies
- an option to see a "passed" state below linked motions

## Design

- [Figma Link - Active stream with funds to claim](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5544%3A601229)
- [Figma Link - Multiple funding sources to claim](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5740%3A333808)
- [Figma Link - Single funding source multiple tokens to claim](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5740%3A334279)
- [Figma Link - Multiple funding sources claiming limit is reached](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5755%3A335130)
- [Figma Link - Single source multiple tokens linked and limit reached](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5755%3A335596)

Active stream with funds to claim
Until - popover 1

Multiple 
Multiple funding 2
funding sources to claim

Single funding source multiple tokens to claim
Single team, two tokens - claim 3

Multiple funding sources claiming limit is reached
Multiple funding 4

Single source multiple tokens linked and limit reached
Single team, two tokens - claim 5

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

This PR adds active stream with funds to claim. UI only.  
After clicking 'Start Stream' there is an active stream with funds to claim. "Paid to date" and "Available to claim" are mocks.
State with 'no funds to claim' and other states (failed, cancelled, ...) will be added in next PRs.

Screenshot 2022-09-22 at 13 31 34

[Figma link](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5544%3A601229)

Resolves #3845

### CODE REVIEW

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.css [hunk 1]
```diff
@@ -93,7 +93,43 @@
   white-space: nowrap;
 }
 
+.value {
+  display: flex;
+  align-items: center;
+  font-size: var(--size-normal);
+  color: var(--dark);
+  letter-spacing: 0.1px;
```
Comment (willm30) [inline src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.css:101]: We tend not to use fractions of a pixel

**Review by willm30 — APPROVED** [review]
LGTM 👍

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx [hunk 2]
```diff
@@ -178,17 +216,109 @@ const StreamingStagesLocked = ({
...
+                style={buttonStyles}
+              />
+            )}
           </div>
         </div>
       </FormSection>
-      <div className={styles.stagesRow}>
+      <div
+        className={classNames(styles.stagesRow, {
+          [styles.alignStart]: paidToDate && paidToDate?.length > 1,
```
Comment (jakubcolony) [inline src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx:231]: Is the optional chaining needed here?

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx [hunk 3]
```diff
@@ -178,17 +216,109 @@ const StreamingStagesLocked = ({
...
+          {status === Status.StartedStream && paidToDate ? (
+            paidToDate.map((paidToDateItem, index) => {
+              const token = colony?.tokens?.find(
+                (tokenItem) => tokenItem.address === paidToDateItem.token,
+              );
+
+              return (
+                <div
+                  className={classNames(styles.value, {
+                    [styles.marginBottom]:
```
Comment (jakubcolony) [inline src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx:247]: Wouldn't this be better solved in CSS with `first-of-type` and `last-of-type`?

**Review by jakubcolony — APPROVED** [review]
Looks good @agnieszkajarosikloj!

**Review by chinins — APPROVED** [review]
Looking good!

**Comment by agnieszkajarosikloj** [comment]
@arrenv I have added fixes. Can you have a look please?

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStages.tsx [hunk 4]
```diff
@@ -1,18 +1,13 @@
-/* eslint-disable no-console */
 import React from 'react';
-import { useParams } from 'react-router';
 import { defineMessages, FormattedMessage } from 'react-intl';
 import classNames from 'classnames';
 
+// import { useParams } from 'react-router';
```
Comment (arrenv) [inline src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStages.tsx:5]: Why do you need this commented out?

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStages.tsx [hunk 5]
```diff
@@ -1,18 +1,13 @@
...
 import Button from '~core/Button';
 import { FormSection } from '~core/Fields';
-import { Tooltip } from '~core/Popover';
-import Icon from '~core/Icon';
-import { useDialog } from '~core/Dialog';
-import { useColonyFromNameQuery } from '~data/generated';
-import CancelStreamingDialog from '~dashboard/Dialogs/CancelStreamingDialog';
 
 import styles from './StreamingStages.css';
+// import { useColonyFromNameQuery } from '~data/generated';
```
Comment (arrenv) [inline src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStages.tsx:10]: Why do you need this commented out?

**Review by arrenv — COMMENTED** [review]
@agnieszkajarosikloj I am unable to test in the same way, as you are not able to add multiple tokens anymore.

7378c949-7bb2-4052-b1ab-b69cb8807120

**Review by arrenv — CHANGES_REQUESTED** [review]
Thank you @agnieszkajarosikloj, very nice. It is working now with multiple tokens.

It looks like we are still getting an alignment issue with the two sections though

f4fe6e0f-4e09-44ee-b79b-dd10c8f69725

**Comment by agnieszkajarosikloj** [comment]
@arrenv I added fix. Can you have a look, please?

**Review by arrenv — CHANGES_REQUESTED** [review]
Thank you @agnieszkajarosikloj, looks good, except it looks like we lost the space. After adding that back in we should be good.

download

**Comment by agnieszkajarosikloj** [comment]
@arrenv I added space. Can you have a look?

**Review by arrenv — APPROVED** [review]
Thank you @agnieszkajarosikloj, looks good to me!

**Pull request merged**